packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -148,7 +151,9 @@ export class OdspDocumentService implements IDocumentService {
         private readonly socketIoClientFactory: () => Promise<SocketIOClientStatic>,
         private readonly cache: IOdspCache,
         hostPolicy: HostStoragePolicy,
+        private readonly epochTracker: EpochTracker,
```

Comment:
(nit) I wonder if it's type to create an interface that has token callbacks, cahce, hostPolicy, epochTracker, and just pass that interface vs. passing many individual pieces one by one. I.e. there seems to be a set of things that every layer needs / passes around, and if so, we should just group them into single object.

Reply:
Makes sense. I will do a separate PR for it.